server/@types/ui/index.d.ts

```diff
@@ -164,3 +164,7 @@ export type LocalAuthority = {
   id: string
   name: string
 }
+
+export type Premises = ApiPremises & {
```

Comment:
I know it's a bit more work but I wonder if this should live in a CAS3-specific type file with its own namespace in the `paths` object in the `tsconfig.json`? 
It seems a bit awkward to have to know that if you're referring to a CAS3 Premises it's in the UI types and a CAS1 Premises it's the api types

Reply:
Ah, so this is an artefact of UI being ahead of API, which now that we've got a second API dev starting I'm hoping will fix itself. Basically as soon as `address` makes its way into API main, this goes away!

Reply:
makes sense - thanks